time timestamp
string box_position
int32 order_no

